Vagrant-setup/travis_create_db_users.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -e
```

Comment:
Can we create _Travis-setup_ folder in project's directory and copy all Travis-related configs over there